chef/cookbooks/hyperv/recipes/openstack_install_neutron.rb

```diff
@@ -25,3 +25,9 @@
   not_if { ::File.exists?("#{node[:openstack][:neutron][:installed]}") }
 end
 
+utils_line "ensure correct python path in shebang in neutron" do
+  file "#{node[:python][:scripts]}\\#{node[:service][:neutron][:name]}-script.py"
+  regexp /\A#!.*/
```

Comment:
Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.